app/models.py

```diff
@@ -31,19 +31,44 @@ class User(Base):
     __tablename__ = "users"
 
 
+class Location(Base):
+    id = Column(Integer, primary_key=True, index=True)
+
+    osm_id = Column(BigInteger)
```

Comment:
why don't we use osm_id as a primary key directly, as it's an INT?

Reply:
I'd rather not rely 100% on OSM. There's probably future locations that will not/never be in OSM. Like online stores, or temporary markets, or even closed locations 🤔...